pkg/image/registry/imagestream/strategy.go

```diff
@@ -74,7 +74,7 @@ func (s Strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList
 	stream := obj.(*api.ImageStream)
 	var errs field.ErrorList
 	if err := s.validateTagsAndLimits(ctx, nil, stream); err != nil {
-		errs = append(errs, field.InternalError(field.NewPath("<none>"), err))
+		errs = append(errs, field.InternalError(field.NewPath(""), err))
```

Comment:
`[]: Internal error: ImageStream "is" is forbidden: exceeded openshift.io/images`